src/jit/assertionprop.cpp

```diff
@@ -3414,6 +3421,26 @@ GenTreePtr Compiler::optAssertionProp_Ind(ASSERT_VALARG_TP assertions, const Gen
 {
     assert(tree->OperIsIndir());
 
+    // TODO-1stClassStructs: All indirections should be handled here, but
+    // previously, when these indirections were GT_OBJ, or implicit due to a block
+    // copy or init, they were not being handled.
+    bool propagateIndir = true;
```

Comment:
Um, no ... I had more complex logic here before, and once I realized that I could simplify it I neglected to remove this. Will delete it.

---
In reply to: [77274805](https://github.com/dotnet/coreclr/pull/7027#discussion_r77274805) [](ancestors = 77274805)